packages/atclient/include/atclient/string_utils.h

```diff
@@ -37,6 +37,16 @@ bool atclient_string_utils_starts_with(const char *string, const char *prefix);
  */
 bool atclient_string_utils_ends_with(const char *string, const char *suffix);
 
+/**
+ *@brief locates the position of a subtring in a given string
```

Comment:
- add space before `@brief`
- `subtring` -> `substring`

Reply:
This change has been made in [2795073](https://github.com/atsign-foundation/at_c/pull/385/commits/279507393dc0f61b49b9e177c30cef390bc3145b)